### PR TITLE
Update webdriver/ and websockets/ from monorail results

### PR DESCRIPTION
**Pull request by stephenmcgruer**
See #798 for the methodology.

### CODE REVIEW

**Review by amberlowh — APPROVED** [review]

**Comment by amberlowh** [comment]
Hi

File: webdriver/tests/delete_cookie/META.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1044449
```
Comment (foolip) [inline webdriver/tests/delete_cookie/META.yml:3]: delete.py is mentioned, but the bug isn't about this failure, so seems not worth linking.

File: webdriver/tests/find_element/META.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/974254
```
Comment (foolip) [inline webdriver/tests/find_element/META.yml:3]: The test was used as an example in this bug, but the bug seems unrelated to the test failure.

File: webdriver/tests/get_active_element/META.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1020018
```
Comment (foolip) [inline webdriver/tests/get_active_element/META.yml:3]: This is about the test being flaky, but yeah, let's link to it, could be relevant.

File: webdriver/tests/get_element_shadow_root/META.yml [hunk 4]
```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1168659
```
Comment (foolip) [inline webdriver/tests/get_element_shadow_root/META.yml:3]: Linking directly to https://bugs.chromium.org/p/chromedriver/issues/detail?id=3715 might be better.

File: webdriver/tests/get_named_cookie/META.yml [hunk 5]
```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1106413
```
Comment (foolip) [inline webdriver/tests/get_named_cookie/META.yml:3]: This bug mentioned the test in a manual import, but isn't about the test but rather something in external/wpt/webmessaging/multi-globals.

File: webdriver/tests/release_actions/META.yml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1128104
```
Comment (foolip) [inline webdriver/tests/release_actions/META.yml:3]: This bug is about why the test was disabled in Chromium. I guess that might be relevant, so even though it doesn't say why it fails in wptrunner, let's link it.

File: webdriver/tests/status/META.yml [hunk 7]
```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1106413
```
Comment (foolip) [inline webdriver/tests/status/META.yml:3]: Same bug as above, let's not link.

File: webdriver/tests/switch_to_frame/META.yml [hunk 8]
```diff
@@ -0,0 +1,5 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1106413
```
Comment (foolip) [inline webdriver/tests/switch_to_frame/META.yml:3]: Same bug as above, let's not link.

File: websockets/cookies/META.yml [hunk 9]
```diff
@@ -0,0 +1,9 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1141909
```
Comment (foolip) [inline websockets/cookies/META.yml:3]: Looks like the right bug, yeah. The bug also mentions 001.html?wss and isn't exactly the same list, but quite close.

File: websockets/unload-a-document/META.yml [hunk 10]
```diff
@@ -0,0 +1,6 @@
+links:
+    - product: chrome
+      url: https://crbug.com/801564
```
Comment (foolip) [inline websockets/unload-a-document/META.yml:3]: The bug title "Implement WebSockets over HTTP/2" makes it seem like a poor match, but the test was tweaked. I feel I'd need to dig deeper to know if it's a relevant bug or not. So let's link it, and unlink if a better one is filed. (Let's ignore that deleting links isn't supported in the UI.)

**Pull request merged**